.changeset/tender-buckets-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/switch": minor
+---
+
+fix(switch): migrate mods
```

Comment:
```suggestion
Migrate `--mod-focus-indicator-thickness` out of the themes to the base index.css. Added a deprecation notice for this mod as it is a duplicate and does not follow naming conventions.
```

@cdransf Could you make your changeset notes a little more readable in future like this? Ideally enough information so that downstream consumers know what actions they will need to take to bring in this change.

Reply:
Absolutely!